conf/nginx.conf

```diff
@@ -66,6 +92,47 @@ http {
       proxy_set_header X-Forwarded-Server $http_host;
       proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
       proxy_set_header X-Request-Start "t=${msec}";
+
+      # The /api/badge endpoint limit is chosen to limit the
+      # load from any single user, and take advantage of latency
+      # not being critical.
```

Comment:
Didn't mention it last time round, but I find these comments are much clearer than before 👍